File: R/Iniciante/1009.r
input <- file('stdin', 'r')

nome <- readLines(input, n = 1)
salario <- as.double(readLines(input, n = 1))
vendas <- as.double(readLines(input, n = 1))

salario = salario + vendas * 0.15

cat(sprintf("TOTAL = R$ %.2f\n", salario))

File: C#/Iniciante/1009.cs
using System; 

class URI {

    static void Main(string[] args) { 

        string nome = Console.ReadLine();
        double salario = Convert.ToDouble(Console.ReadLine());
        double vendas = Convert.ToDouble(Console.ReadLine());
        vendas *= 0.15;
        
        salario += vendas;
        
        Console.WriteLine($"TOTAL = R$ {salario:F2}");

    }

}
